app/src/pages/playground/playgroundUtils.ts

```diff
@@ -1,20 +1,28 @@
-import { generateInstanceId, PlaygroundInstance } from "@phoenix/store";
+import { PlaygroundInstance } from "@phoenix/store";
 import {
+  ChatMessage,
   ChatMessageRole,
-  chatMessageRoles,
+  createPlaygroundInstance,
   generateMessageId,
 } from "@phoenix/store/playgroundStore";
 import { safelyParseJSON } from "@phoenix/utils/jsonUtils";
 
 import { ChatRoleMap, DEFAULT_CHAT_ROLE } from "./constants";
-import { llmAttributesSchema } from "./schemas";
+import {
+  chatMessageRolesSchema,
+  chatMessagesSchema,
+  llmInputMessageSchema,
+  llmOutputMessageSchema,
+  MessageSchema,
+  outputSchema,
+} from "./schemas";
 import { PlaygroundSpan } from "./spanPlaygroundPageLoader";
 
 /**
  * Checks if a string is a valid chat message role
  */
 export function isChatMessageRole(role: unknown): role is ChatMessageRole {
-  return chatMessageRoles.includes(role as ChatMessageRole);
+  return chatMessageRolesSchema.safeParse(role).success;
```

Comment:
we don't have to use this but i wanted to add it to the shcmea for ChatMessages so figured would update here as well